_layouts/formula.html

```diff
@@ -30,41 +30,54 @@
 <p><a target="_blank" href="{{ site.taps.linux.remote }}/blob/HEAD/Formula/{{ f.name }}.rb">Linux formula code</a> on GitHub</p>
 {%- else %}
 <p><a target="_blank" href="{{ site.taps.core.remote }}/blob/HEAD/Formula/{{ f.name }}.rb">Formula code</a> on GitHub</p>
+{%- endif -%}
+
+{%- if f.versions.bottle %}
+<p>Bottle support{%- unless formula_path == "formula-linux" %} for macOS releases{%- endunless -%}:</p>
+<table>
+    <tr>
+        <td><strong>Intel</strong></td>
```

Comment:
This is also macOS specific.

Reply:
Could also be:
```suggestion
        <td><strong>Intel x86_64</strong></td>
```

or just

```suggestion
        <td><strong>x86_64</strong></td>
```